pom.xml

```diff
@@ -111,6 +111,7 @@
         <jupiter.version>5.11.0</jupiter.version>
         <hamcrest.version>2.2</hamcrest.version>
         <maven-failsafe-plugin.version>3.0.0-M7</maven-failsafe-plugin.version>
+        <maven-surefire-plugin.version>3.1.2</maven-surefire-plugin.version>
```

Comment:
Why are you using different versions for failsafe and surefire?

Reply:
nothing intentional. I have just use some version and didn't want to change more than it is needed. But as you pointed it out I have updated also failsafe plugin version to match surefire. Thanks